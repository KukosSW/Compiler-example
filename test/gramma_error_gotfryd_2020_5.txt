DECLARE
  a
BEGIN
  WRITE a;
END
