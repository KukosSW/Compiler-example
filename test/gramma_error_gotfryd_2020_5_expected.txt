BLAD: Uzycie zmiennej a bez inicjalizacji w linii 4
